### ISSUE TITLE
Accessibilty issues.

### ISSUE THREAD

**Issue by PixeledCode** [ibrahimcesar/react-lite-youtube-embed#21]
**Describe the bug**
Currently, as per my testing, the div generated by the plugin is not focusable using the tab key. That means they are not accessible by screen-readers or keyboard users.

**To Reproduce**
1. Go to https://react-lite-youtube-embed.ibrahimcesar.cloud/
2. click on the `Tab` key on the keyboard. 

**Expected behavior**
The div should get focus and on `Enter`, it should fetch the `iFrame` and play the video

**Possible Fix**
Changing the youtube play button to a `button` from `div` seems to fix it.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome, Mozilla

**Comment by ibrahimcesar** [comment]
Hi, @ibrahimcesar thanks for the update. It does make it focusable by keyboard users but it's still not accessible. 

I'm using NVDA for testing and it declares the `button` like 'button'. That's very hard for them to know what button is it.

Officially, Youtube does it like, 'Watch {Video Title} link'

So here's what I suggest:

 - For the `div`, we can remove `tabindex` since it's making video focusable 2 times.
 - For the Button, add an `aria-label` of `Watch, {titleName}`

So when a user focuses on the video, it will announce it as, "clickable watch, video title, button"

Let me know what you think and thanks a lot for taking your time to improve accessibility of the tool.

_Originally posted by @PixeledCode in https://github.com/ibrahimcesar/react-lite-youtube-embed/issues/22#issuecomment-904286994_

**Comment by ibrahimcesar** [comment]
Hello @PixeledCode , I just published a new version with your points. When you could find some time, I deployed the new approach at the [demo page](https://react-lite-youtube-embed.ibrahimcesar.cloud/). Thanks for all the help!

**Comment by PixeledCode** [comment]
> Hello @PixeledCode , I just published a new version with your points. When you could find some time, I deployed the new approach at the [demo page](https://react-lite-youtube-embed.ibrahimcesar.cloud/). Thanks for all the help!

Thanka a lot @ibrahimcesar. Really appreciate you taking the time and effort to making the tool more accessible.

**Comment by ibrahimcesar** [comment]
@PixeledCode Luck you point me out! I had to thank you for help me build a better tool for everyone! Thanks!
